pos_margin_stored/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2025 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "PoS Order Margin",
+    "summary": "Margin Stored on PoS Order",
+    "version": "16.0.1.0.0",
+    "category": "Point Of Sale",
+    "author": "GRAP, Odoo Community Association (OCA)",
+    "maintainers": ["quentinDupont"],
+    "website": "https://github.com/OCA/pos",
+    "license": "AGPL-3",
+    "depends": [
+        "pos_margin",
+    ],
```

Comment:
```suggestion
    "depends": ["point_of_sale"],
```

I think the dependency to the OCA module is not necessary. Don't you think ?

Reply:
you're god damn right !